CHANGELOG.md

```diff
@@ -1,23 +1,18 @@
 For 2.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 
-1.20.0 (in progress)
-====================
+1.20.0 (November 11, 2019)
+==========================
 
 - twilio-video.js will now support the Unified Plan SDP format for Google Chrome.
```

Comment:
As of this release, twilio-video.js will no longer use the deprecated Plan B SDP format when publishing or subscribing to tracks. It will use the [Unified Plan](https://tools.ietf.org/html/draft-roach-mmusic-unified-plan-00) format. Google has advised that they will remove Plan B support from Chrome during Q1 2020. Therefore, we recommend updating to SDK 1.20.0+ as soon as possible. This change will not impact interoperability with existing twilio-video.js versions or other supported versions.

Reply:
The referenced doc seems to be outdated, has expiration date of 2014. We may reference this one, it's much newer - https://webrtc.org/web-apis/chrome/unified-plan/ @makarandp0

Reply:
Please update the unified plan link with https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-26

Reply:
Thanks @aymenn - I have incorporated your suggestions. It reads much better now.